### ISSUE TITLE
StaticInjectorError[MzToastService]

### ISSUE THREAD

**Issue by Liad91** [sherweb/ng2-materialize#248]
Getting to following error after update to 1.7.1
```
Error: StaticInjectorError[MzToastService]: 
  StaticInjectorError[MzToastService]: 
  NullInjectorError: No provider for MzToastService!
```

## Your Environment
* Version used: 1.7.1
* Angular version used: 4.4.6 / 5.0.3
* Browser Name and version: chrome
* Operating System and version (desktop or mobile): desktop
* Link to your project (if appropriate): https://github.com/Liad91/VeganNearby

**Comment by jfcere** [comment]
@Liad91 Thank for the PR!

**Comment by jfcere** [comment]
@Liad91 your fix has been released with [v1.7.2](https://github.com/sherweb/ng2-materialize/releases/tag/v1.7.2)
